deploy/FlapperInit.sol

```diff
@@ -143,22 +150,41 @@ library FlapperInit {
         dss.chainlog.setAddress(clKey, wrapper_);
     }
 
+    function initFarm(
+        DssInstance memory dss,
+        address            farm_,
+        FarmConfig  memory cfg
+    ) internal {
+        FarmLike     farm     = FarmLike(farm_);
+        SplitterLike splitter = SplitterLike(cfg.splitter);
+
+        require(farm.rewardsToken() == DaiJoinLike(cfg.daiJoin).dai(), "Farm rewards not dai");
```

Comment:
we should probably fetch `daiJoin` from the splitter and remove it from the `cfg`.

Reply:
And I guess this raises the question whether we also assume the splitter must also be init before the flapper, since in that case we can do the same in `initFlapperUniV2`. Maybe that for that we can leave it as is, since we don't gain that much by enforcing the order.

Reply:
Hmm yeah not sure what to do here, not really strong opinion, but the lack of symmetry will look weird.

Reply:
ok, I don't feel strong about it, can leave as is.

Reply:
Ok good let's leave this way then